core/src/test/scala/unit/kafka/utils/CoreUtilsTest.scala

```diff
@@ -183,31 +185,26 @@ class UtilsTest extends JUnitSuite {
   }
 
   @Test
-  def testGetOrElseUpdateAtomically(): Unit = {
+  def testAtomicGetOrUpdate(): Unit = {
     val count = 1000
     val nThreads = 5
     val createdCount = new AtomicInteger
     val map = new ConcurrentHashMap[Int, AtomicInteger]().asScala
-    val executor = Executors.newFixedThreadPool(nThreads)
+    implicit val executionContext = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(nThreads))
```

Comment:
We create the execution context because we need to control the number of threads, but normally we can use the global executor context and that is even more concise. Especially in tests, there's not much reason to use old style runnables.